configure.ac

```diff
@@ -4,11 +4,11 @@ define(_CLIENT_VERSION_MAJOR, 1)
 define(_CLIENT_VERSION_MINOR, 14)
 define(_CLIENT_VERSION_REVISION, 7)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
-define(_COPYRIGHT_YEAR, 2022)
+define(_CLIENT_VERSION_IS_RELEASE, true)
+define(_COPYRIGHT_YEAR, 2024)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
-define(_CURRENT_RELEASE_DATE,[[1970-01-01]])
+define(_CURRENT_RELEASE_DATE,[[2024-02-27]])
 define(_EXPECTED_DAYS_TO_NEXT_RELEASE,[270])
```

Comment:
I like this for us.

Reply:
We're so screwed 🤣

Reply:
just curious, does the `_EXPECTED_DAYS_TO_NEXT_RELEASE` show up anywhere else?

Reply:
A post-American-Thanksgiving release!

Reply:
> just curious, does the `_EXPECTED_DAYS_TO_NEXT_RELEASE` show up anywhere else?

See #3191 and #3161 for where this came from and where we intend to use it.

Reply:
E A P E A F

Reply:
PK PF